account_payment_order/views/account_payment_order.xml

```diff
@@ -105,6 +107,7 @@
                 <filter name="date_generated_groupby" string="File Generation Date" context="{'group_by': 'date_generated'}"/>
                 <filter name="date_uploaded_groupby" string="File Upload Date" context="{'group_by': 'date_uploaded'}"/>
                 <filter name="state_groupby" string="State" context="{'group_by': 'state'}"/>
+                <filter name="description_groupby" string="Description" context="{'group_by': 'description'}"/>
```

Comment:
@GillesTephaneMeyomesse You only need to add `<field name="description"/>` in the search view

Reply:
the groupby on the description isn't necessary